dev-app-post/main/default/profiles/Admin.profile-meta.xml

```diff
@@ -335,6 +370,176 @@
         <field>Case.Verbal_Order__c</field>
         <readable>true</readable>
     </fieldPermissions>
+    <fieldPermissions>
+        <editable>true</editable>
+        <field>Case.EDRD_Additional_information__c</field>
+        <readable>true</readable>
+    </fieldPermissions>
+	<fieldPermissions>
+        <editable>true</editable>
+        <field>Case.EDRD_Coverage_Status__c</field>
+        <readable>true</readable>
+    </fieldPermissions>
+	<fieldPermissions>
+        <editable>true</editable>
+        <field>Case.EDRD_Annual_Max_Coverage__c</field>
+        <readable>true</readable>
+    </fieldPermissions>
+	<fieldPermissions>
+        <editable>true</editable>
+        <field>Case.EDRD_Dosage__c</field>
+        <readable>true</readable>
+    </fieldPermissions>
+	<fieldPermissions>
+        <editable>true</editable>
+        <field>Case.EDRD_Drug_cost_forecast__c</field>
```

Comment:
EDRD Drug cost forecast needs to be changed to EDRD Drug Cost Forecast @rahulshyamrajsingh

Reply:
The field's label has been changed because the build is giving us issues when we attempt to modify the API name.